backend/app/services/llm/jobs.py

```
@@ -226,6 +259,17 @@ def execute_job(
         )
 
         if response:
+            if output_guardrails:
+                output_text = response.response.output.text
+                safe_output = call_guardrails(output_text, output_guardrails, job_id)
+                logger.info(
+                    f"[execute_job] Output guardrail validation | Original output={output_text}, safe_output={safe_output}/"
+                )
+                if safe_output["success"] == True:
```

Ruff (0.14.14)

[error] Avoid equality comparisons to `True`; use `safe_output["success"]:` for truth checks

Replace with `safe_output["success"]`

(E712)